packages/adp-tooling/src/types.ts

```diff
@@ -175,6 +229,7 @@ export type IWriterData<T extends ChangeType> = IWriter<GeneratorData<T>>;
  *
  * @template T - The specific type of data the writer will handle, determined by the associated ChangeType.
  */
+
```

Comment:
Please remove this extra empty line as it detaches the jsdoc from the interface.

Reply:
Removed in: [4e27dbf](https://github.com/SAP/open-ux-tools/pull/1775/commits/4e27dbff0ba748cc271d6d99bb08a05d7657dd4a)